package.json

```diff
@@ -118,6 +118,7 @@
       "jsx-a11y/no-static-element-interactions": "off",
       "react/forbid-prop-types": "off",
       "operator-linebreak": "off",
+      "object-curly-newline": "off",
```

Comment:
I've add this for this line : https://github.com/alakarteio/k-ramel/pull/191/files#diff-302ebb27e4b4869517b091be43c39c92R18